lib/dust.js

```diff
@@ -725,6 +725,11 @@
     return this;
   };
 
```

Comment:
I added these docs straight from http://www.dustjs.com/docs/helper-api/.  Looks like other docs are missing from some chunk methods that are on the helper-api page.  I could add these if they aren't considered bloat.  This Chunk#map method is great and the docs really made it's usage clear to me.  As I was looking at making the code smaller / more resuable, it seemed apparent that what I really needed was a new map method for thenables, so I created the mapThenable(...) method.  I'd like to put it on the Chunk object, but that would be altering the public API of that and I'm not sure that desirable.  I'm also wondering if I should move the Chunk#renderError method off of Chunk for the same reason or if it's simple, generic, and useful enough to keep.